pyproject.toml

```diff
@@ -118,8 +124,10 @@ exclude_also = [
 
 [tool.pytest.ini_options]
 addopts = [
+    "--import-mode=importlib",
     "--strict-markers",
     "--doctest-modules",
+    "--pyargs",
```

Comment:
I'm not sure I totally get this argument. What does this do other than let us use `anndata` instead of `./src/anndata` in `testpaths`?

Reply:
because doctests are weird and would otherwise result in importing some modules as files and others via `sys.path`.